adapters/Mintegral/MintegralAdapter/Bidding/GADMAdapterMintegralInterstitialRenderer.m

```diff
@@ -0,0 +1,151 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "GADMAdapterMintegralInterstitialRenderer.h"
+#import "GADMAdapterMintegralUtils.h"
+#import "GADMediationAdapterMintegralConstants.h"
+#import "GADMAdapterMintegralExtras.h"
+#include <stdatomic.h>
+#import <MTGSDKNewInterstitial/MTGSDKNewInterstitial.h>
+#import <MTGSDKNewInterstitial/MTGNewInterstitialBidAdManager.h>
+
+@interface GADMAdapterMintegralInterstitialRenderer ()<GADMediationInterstitialAd,MTGNewInterstitialBidAdDelegate>
+
+@end
+
+@implementation GADMAdapterMintegralInterstitialRenderer
+{
+    // The completion handler to call when the ad loading succeeds or fails.
+    GADMediationInterstitialLoadCompletionHandler _adLoadCompletionHandler;
+    
+    /// Data used to render an interstitial Ad.
+    GADMediationInterstitialAdConfiguration *_adConfiguration;
+    
+    // The mintegral interstitial ad.
+    MTGNewInterstitialBidAdManager *_interstitialAd;
+    // An ad event delegate to invoke when ad rendering events occur.
+    // Intentionally keeping a reference to the delegate because this delegate is returned from the
+    // GMA SDK, not set on the GMA SDK.
```

Comment:
We can remove this comment, or can you clarify why is this added for interstitial ad?

Reply:
ok,I will delete